src/main/java/Duke.java

```diff
@@ -8,21 +8,39 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
 
-        String line = "_________________________________\n";
+        String line = "____________________________________________________\n";
         System.out.println("Hello from\n" + logo);
         System.out.print(line + "Hello! I'm Duke\n" + "What can I do for you?\n" + line);
         String[] list = new String[100];
+        boolean[] isDone = new boolean[100];
+        String toFill;
         int listCount = 0;
         while(true) {
             Scanner userInput = new Scanner(System.in);
             String input = userInput.nextLine();
             if (input.equals("bye")) {
                 System.out.print(line + "Bye. Hope to see you again soon!\n" + line);
                 break;
-            } else if(input.equals("list")) {
+            } else if (input.startsWith("mark")) {
+                int mark = Integer.parseInt(input.substring(5));
+                isDone[mark - 1] = true;
+                System.out.println(line + "Nice! I've marked this task as done:\n" + "  [X] " + list[mark - 1]);
+                System.out.print(line);
+            } else if (input.startsWith("unmark")) {
+                int unmark = Integer.parseInt(input.substring(7));
+                isDone[unmark - 1] = false;
+                System.out.println(line + "OK, I've marked this task as not done yet:\n" + "  [ ] "+ list[unmark - 1]);
                 System.out.print(line);
+            } else if(input.equals("list")) {
```

Comment:
Good indentation and presenting the conditional statements, which helps the code's readability.